### PR TITLE
Keyword extraction api test cases

### PR DESCRIPTION
**Pull request by nidhiya-pious**

### CODE REVIEW

File: cypress/e2e/KeyWordExtraction/apiInfo.cy.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    });
+
+    it('3. Test to check for the title "What is Keyword Extraction?"', () => {
+        cy.checkTextVisibility('What is Keyword Extraction?');
+    });
+
+    it('4. Test to check for the title "Use cases for Keyword Extraction API"', () => {
+        cy.checkTextVisibility('Use cases for Keyword Extraction API');
+    });
+
```
Comment (abhishek9celestial) [inline cypress/e2e/KeyWordExtraction/apiInfo.cy.js:27]: Can you also check the text for "How it works".

File: cypress/e2e/KeyWordExtraction/documentation.cy.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+import { KEYWORD_EXTRACTION_BASE_URL} from '../../resources/data';
+
+describe('Keyword Extraction documentation page', () => {
```
Comment (abhishek9celestial) [inline cypress/e2e/KeyWordExtraction/documentation.cy.js]: Can you make it as Documentation.

File: cypress/e2e/KeyWordExtraction/documentation.cy.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    });
+
+    it('2. Test to click on the "Documentation" tab', () => {
+        cy.get('#documentation-tab').click();
+        cy.checkTextVisibility('Keyword Extraction API Reference');
+    });
+
+    it('3. Test to check and click link "development quickstart guide."', () => {
+        cy.contains('a', 'development quickstart guide').should('be.visible');
+        cy.contains('a', 'development quickstart guide').click();
```
Comment (abhishek9celestial) [inline cypress/e2e/KeyWordExtraction/documentation.cy.js:21]: How you are navigating back to the page after clicking the link.

File: cypress/e2e/KeyWordExtraction/documentation.cy.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    });
+
+    it('12. Test to click on the "Error Codes" link', () => {
+        cy.get('li > a').contains('Error Codes').click();
+        cy.get('[name="errors"]').should('be.visible');
+    });
+
+    it('13. Test to click on "Accounts Pages" link', () => {
+        cy.contains('a', 'Accounts page').should('be.visible');
+        cy.contains('a', 'Accounts page').click();
```
Comment (abhishek9celestial) [inline cypress/e2e/KeyWordExtraction/documentation.cy.js:68]: Same here also

File: cypress/e2e/KeyWordExtraction/documentation.cy.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    before(() => {
+        cy.visit(KEYWORD_EXTRACTION_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });
+
+    it('2. Test to click on the "Documentation" tab', () => {
+        cy.get('#documentation-tab').click();
```
Comment (NaveenK15) [inline cypress/e2e/KeyWordExtraction/documentation.cy.js:14]: API request interupt

**Pull request merged**